src/GovUk.Education.ExploreEducationStatistics.Data.Processor/GovUk.Education.ExploreEducationStatistics.Data.Processor.csproj

```diff
@@ -27,5 +27,9 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <CopyToPublishDirectory>Never</CopyToPublishDirectory>
     </None>
+    <None Update="appsettings.Local.json">
```

Comment:
Was expecting to see the equivalent change in Content API and Admin's csproj files too

Reply:
If we don't add this, `appsettings.Local.json` doesn't get included in the build output. 

The standard .NET projects seem to implicitly bundle appsettings files so we don't need these changes in those projects.

However, the Function projects seem to be more aggressive in stripping out unnecessary output files hence needing this.

Reply:
Aha OK makes sense - Function project bundling strikes again!  Thanks for clarifying